main.go

```diff
@@ -223,9 +240,9 @@ func buildDockerFlags() ([]string, error) {
 		dockerGp = append(dockerGp, dv)
 	}
 	res = append(res,
-		"--env=GOPATH="+strings.Join(dockerGp, string(os.PathListSeparator)),
+		"-e", "GOPATH="+strings.Join(dockerGp, string(os.PathListSeparator)),
```

Comment:
briefly document that we use -e instead of --env for the sake of docker-compose

Reply:
Good point thanks.